roles/pulp-devel/templates/alias.bashrc.j2

```diff
@@ -163,12 +163,40 @@ pbindings(){
     if [ $1 != 'pulpcore' ]
     then
         sudo ./generate.sh pulpcore $2
-        pip install ./pulpcore-client
+        case $2 in
+            python)
+                pip install ./pulpcore-client
+                ;;
+            ruby)
+                cd pulpcore-client
+                gem build pulpcore_client
+                GEM_FILE="$(ls | grep pulpcore_client-)"
+                gem install --both ./${GEM_FILE}
+                cd ..
+                ;;
+            *)
+                echo "Sorry, language not supported"
+                ;;
+        esac
         sudo rm -rf pulpcore-client
     fi
 
     sudo ./generate.sh $@
-    pip install ./$1-client
+    case $2 in
+        python)
+            pip install ./$1-client
+            ;;
+        ruby)
+            cd $1-client
+            gem build $1_client
```

Comment:
Are these a safe way to concatenate strings to these bash special variables? I know that bash variables can at least have underscores in them, but special vars (like those beginning with numbers) may follow different rules. If not safe, add the curly brackets.

Reply:
I tested locally(fedora 30 box) and it worked